src/app/theme/panel.css

```diff
@@ -20,6 +20,16 @@
 	margin-bottom: var(--color-border-default) !important;
 }
 
+/* After updating ckeditor to 35.2.1 the update to the balloon panel arrow styling for the dark theme was needed. See #377. */
+html[data-color-mode="dark"] .ck.ck-balloon-panel[class*=arrow_n]:before {
+	margin-top: inherit;
+}
+
+html[data-color-mode="dark"] .ck.ck-balloon-panel[class*=arrow_s]:before {
+	margin-bottom: inherit;
+}
```

Comment:
Having a 1px difference in `margin-top` (or `-bottom`) seems to make sense in all tooltips. From what I see it was designed that way. So in this case I'd prefer not to change this property.

Actually, what I'd suggest is to change the `color` instead of `margin`. We could change it for `arrow_n:before` and `arrow_s:before` the similar way the `:after` is changed here: [https://github.com/ckeditor/github-writer/blob/c84b7421886e03cf05e8beb1df6242354e48dde9/src/app/theme/panel.css#L13-L21](https://github.com/ckeditor/github-writer/blob/c84b7421886e03cf05e8beb1df6242354e48dde9/src/app/theme/panel.css#L13-L21)

What's really interesting here in that permalink I just pasted is that there are bugs, because `margin-top` and `margin-bottom` are set with `var(--color-border-default)` which does not make sense and results in `margin` to be simply `0` (which makes the triangle border look thiner and we don't want that).

Solution: I think we should remove these incorrect margins first, and then add two rules for `arrow_n:before` and `arrow_s:before` and set appropriate border color. I'd go with the same color as tooltip border color, i.e. `var(--color-border-default)`.

Reply:
Oh, and btw, if we go with this color change (instead of margin), we avoid rules that depend on theme name e.g. we don't have to write something like this `html[data-color-mode="dark"]` anymore.